scripts/gis-material-update/validate-deploy/modules/hsl.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from modules.config import Config
+from modules.common import *
+from modules.gis_validate_deploy import GisProcessor
+
+class HslBuses(GisProcessor):
+    """Process HSL bus lines."""
+
+    def __init__(self, cfg: Config):
+        self._cfg = cfg
+        self._module = "hsl"
+        self._tormays_table_org = self._cfg.tormays_table_org(self._module)
+        self._tormays_table_temp = self._cfg.tormays_table_temp(self._module)
+        self._validate_limit_min = self._cfg.validate_limit_min(self._module)
+        self._validate_limit_max = self._cfg.validate_limit_max(self._module)
+        self._pg_conn_uri = self._cfg.pg_conn_uri()
+        logging.basicConfig(
+            filename=self._cfg.logging_filename().format(self._module), 
+            filemode=self._cfg.logging_filemode(), 
+            format=self._cfg.logging_format()
+            )
+        self._logger = logging.getLogger()
+        self._logger.setLevel(self._cfg.logging_level())
+                            
+    def validate_deploy(self):
+
+        # validate data amount: is it between given limits
+        self._validate_result = validate_data_count_limits(
+            self._module,
+            self._pg_conn_uri, 
+            self._tormays_table_org, 
+            self._tormays_table_temp, 
+            self._validate_limit_min, 
+            self._validate_limit_max, 
+            self._logger
+            )
+        
+        # Valid --> deploy
+        if self._validate_result == "Valid":
+            self._deploy_result = deploy(
+                self._pg_conn_uri, 
+                self._tormays_table_org, 
+                self._tormays_table_temp, 
+                self._logger
+                )
```

Comment:
I think the temp table should be deleted after the deployment is completed. This could be done as a separate function call here since it's not critical if it fails. But we want to clean up after ourselves, and not leave the production database with any unnecessary tables.

Reply:
This part is changed. No unnecessary tables anymore.